index.html

```diff
@@ -4,8 +4,195 @@
+              <p class="icon-description">
+                Listen to the speakers from various countries about iranian
+                cinema
+              </p>
+            </article>
+          </li>
+          <li>
+            <article class="program-parts">
+              <img class="icon" src="./assets/exhibition.png" alt="" />
+
+              <h4 class="icon-title">Exhibition</h4>
+              <p class="icon-description">
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+                Incidunt
+              </p>
+            </article>
+          </li>
+          <li>
+            <article class="program-parts">
+              <img class="icon" src="./assets/chat-group.png" alt="" />
+
+              <h4 class="icon-title">Forum</h4>
+              <p class="icon-description">
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+                Incidunt
+              </p>
+            </article>
+          </li>
+          <li>
+            <article class="program-parts">
+              <img class="icon" src="./assets/training.png" alt="" />
+              <h4 class="icon-title">Workshop</h4>
+              <p class="icon-description">
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+                Incidunt
+              </p>
+            </article>
+          </li>
+          <li>
+            <article class="program-parts">
+              <img class="icon" src="./assets/networking.png" alt="" />
+              <h4 class="icon-title">Networking</h4>
+              <p class="icon-description">
+                Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+                Incidunt
+              </p>
+            </article>
+          </li>
+        </ul>
+        <button class="btn-join" id="see-program-btn">
+          Join the program now!
+        </button>
+        <a href="#" class="see-the-program desktop-only" id="see-program-link"
+          >See the whole program</a
+        >
+      </section>
+      <section class="speakers" id="news">
+        <!-- This section will create dynamically via script.js when the page loads. -->
+      </section>
+      <section class="desktop-only">
+        <section class="partners-container" id="sponsor">
+          <h3 class="main-program-title">Partner</h3>
+          <hr class="speakers-line" />
+          <section class="partners">
+            <img src="./assets/imdb.png" alt="" />
+            <img src="./assets/oscars.png" alt="" />
+            <img src="./assets/cann.png" alt="" />
+            <img src="./assets/bbc.png" alt="" />
+          </section>
+        </section>
+
+        <section class="footer-home">
+          <img
+            src="./assets/logo-black.png"
+            alt=""
+            class="footer-home-logo"
+          />
+          <h2 class="footer-home-text">
+            Lorem ipsum dolor sit amet consectetur<br />Lorem ipsum dolor sit
```

Comment:
- [ ] Kindly make to have the same content in the footer as the one you have in the `about` page footer in order for your website to look professional. :wink: